supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -33,8 +33,10 @@ Use the following template. Replace _<software_name>_ with the software name:
 ----
 [IMPORTANT]
 ====
-_<software_name>_ is Developer Preview software only. Developer Preview software is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this software to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
-
+_<software_name>_ is Developer Preview software only, and as such, it is not intended for production or business-critical workloads. Developer Preview software is not supported by Red{nbsp}Hat in any way and is not functionally complete.
```

Comment:
Right now, this is rendering as each of these lines on separate lines (unlike normal paragraphs where this would have been smushed together). 

So if it's supposed to be a single block paragraph, then you likely need to put these all on the same line.

But also should consider whether there should be any blank lines between them. At the very least, there should likely be a blank line before "For more information about..." to match how it was in the previous snippet text. But I think it might be a good time to decide whether any additional breaks could help readability